cmd/main.go

```diff
@@ -150,6 +152,23 @@ func main() {
 			os.Exit(1)
 		}
 	} else {
+		metricsCertPath := "/etc/lws/metrics/certs"
```

Comment:
IMO it is better to use the default values and watcher that are provided by the  `metricsserver.Options` . Please see https://github.com/kubernetes-sigs/jobset/pull/863 or more specifically

https://github.com/kubernetes-sigs/jobset/pull/863/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R169

Reply:
According to here https://github.com/kubernetes-sigs/kubebuilder/blob/fa1d379fba769367def698662517f40e985d7c0d/pkg/plugins/golang/v4/scaffolds/internal/templates/cmd/main.go#L361-L381, this is the recommended way. Maybe `metricsserver.Options` does not rotate?

Reply:
@kannon92 If we mount default paths defined in https://github.com/kubernetes-sigs/controller-runtime/blob/76067279145e76148cf27c01ea4ec63fed8e463b/pkg/metrics/server/server.go#L85-L102, we don't need to manually watch certificates. Controller-runtime manages this for us https://github.com/kubernetes-sigs/controller-runtime/blob/76067279145e76148cf27c01ea4ec63fed8e463b/pkg/metrics/server/server.go#L286-L309. As a result, there is no need to have code change in main.go